src/main/java/io/confluent/connect/elasticsearch/Mapping.java

```diff
@@ -56,43 +56,27 @@
    * @param schema The schema used to infer mapping.
    * @throws IOException from underlying JestClient
    */
-  public static void createMapping(JestClient client, String index, String type, Schema schema)
+  public static void createMapping(ElasticsearchClient client,
+                                   String index,
```

Comment:
indentation style is not the one we use in connectors.

Reply:
Hmm, reformatting code with new coding style left this as is.

Reply:
Frequently, to apply the actual style, you need to put in one line, select the whole line, and press Alt+Cmd+L

Reply:
When I do that, it just leaves it all on one line.  What formatting should it have?

Reply:
In IntelliJ, checking the settings under: _Preferences -> Code Style -> Java_
tab _Wrapping and Braces_ 
boxes for _Method alignment parameters_ and _Method call arguments_
and make sure _New line after '('_ is ticked, _Place ')' on new line_ is ticked (when that's the case I believe _Align when multiline_ does not matter, but better let it unticked too).

Reply:
Got it, thanks